R/load-ace-constant-columns.R

```diff
@@ -206,9 +205,24 @@ standardize_ace_column_names <- function(df) {
                   new == "task_switch_state" ~ "taskswitch_state",
                   TRUE ~ new) # for cross compatibility b/w emailed and pulvinar)
   names(df) = new
+
+  # With newer data formats (nexus), "PID" may already be included alongside
+  # an existing "participant_id". In this case, do not map "participant_id"
+  # to "PID" as we'll be creating a duplicate
+  rename_conditionally(df, "participant_id", COL_PID)
+  rename_conditionally(df, "user_id", COL_PID)
+  
   return (df)
 }
 
+rename_conditionally <- function(df, old_name, new_name) {
+  if (old_name %in% names(df) && !(new_name %in% names(df))) {
+    df <- df %>% rename(!!new_name := !!sym(old_name))
+  }
+  df
+}
+
+
```

Comment:
This new function looks very handy! Would you mind moving this over to utils-standardize.R? I think it fits over there as `standardize_names()` is also defined over there.

Reply:
Will do!